app/matplotlib.ipynb

```diff
@@ -339,18 +344,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
-   "outputs": [],
    "source": [
-    "# write your code here\n"
+    "df = pd.DataFrame(pd.read_csv(\"../data/customer_data.csv\"))\n",
+    "\n",
+    "\n",
+    "plt.scatter(df[\"Annual Income\"], df[\"Spending Score\"], color=\"red\" )\n",
+    "plt.scatter(df[\"Annual Income\"], df[\"Age\"], color=\"blue\")\n",
+    "\n",
+    "\n",
+    "plt.show()"
```

Comment:
In Task 9, you are required to create a scatter plot matrix to visualize pairwise relationships between multiple variables. Consider using `pd.plotting.scatter_matrix()` to create a scatter plot matrix instead of individual scatter plots.